inference-engine/thirdparty/clDNN/src/program.cpp

```diff
@@ -144,7 +144,7 @@ void program::init_primitives() {
 }
 
 void program::compile() {
-    _kernels_cache->build_all();
+    _kernels_cache->build_all(prog_id);
```

Comment:
Since kernels_cache instance is created per program, I think this prog_id should be passed in kernels_cache c-tor

Reply:
I thought about it to make kernels_cache to have its own prog_id. 
But just did like this since it seeemed only needed for dump. 
However I have no objection to make kernels_cahce to have own id. 
(It used to be there, but removed at the refactoring PR) 
I will update it.

Reply:
Updated